presto-verifier/src/main/java/com/facebook/presto/verifier/Verifier.java

```diff
@@ -175,6 +175,17 @@ public int run(List<QueryPair> queries)
         return failed;
     }
 
+    private boolean isCheckCpu(QueryPair query)
```

Comment:
Sorry, should have been more clear. I meant that all this code should be moved to `HumanReadableEventClient`